python/test/unit/fem/test_custom_assembler.py

```diff
@@ -366,7 +366,10 @@ def test_custom_mesh_loop_rank1():
         end = time.time()
         print("Time (numba/cffi, pass {}): {}".format(i, end - start))
     b3.vector.ghostUpdate(addv=PETSc.InsertMode.ADD, mode=PETSc.ScatterMode.REVERSE)
-    assert (b3.vector - b0.vector).norm() == pytest.approx(0.0)
+    if dolfin.default_scalar_type in (np.float32, np.complex64):
+        assert (b3.vector - b0.vector).norm() == pytest.approx(0.0, 1e-8)
```

Comment:
Could you make the tolerances a named argument so it's clear that it's the absolute tolerance?